figures/scripts/fig2-oncoprint-landscape.R

```diff
@@ -180,10 +182,18 @@ cancer_group_palette <- group_palette_df %>%
 cancer_group_colors <- cancer_group_palette$oncoprint_hex
 names(cancer_group_colors) <- cancer_group_palette$cancer_group
 
+# For coloring male and female:
+binary_palette_df <- readr::read_tsv(
+  file.path(palette_dir, "binary_color_palette.tsv")
+)
+
+# this variable is used again to make the "Other CNS" legend
+germline_sex_estimate_colors <-  c("Male" = binary_palette_df$hex_codes[binary_palette_df$color_names == "binary_1"],
+                                   "Female" = binary_palette_df$hex_codes[binary_palette_df$color_names == "binary_2"])
```

Comment:
This is not an improvement if we wanted to change the germline sex colors in the OncoPrints independently of the binary palette, which I could very well see happening.

Having a vector of `germline_sex_estimate_colors` that gets used both places certainly would be an improvement.

Reply:
no longer read in binary palette; now we just define the vector and use it